release/NativeEditPlugin/scripts/NativeEditBox.cs

```diff
@@ -319,6 +330,7 @@ private void CreateNativeEdit()
 		jsonMsg["placeHolderColor_b"] = mConfig.placeHolderColor.b;
 		jsonMsg["placeHolderColor_a"] = mConfig.placeHolderColor.a;
 		jsonMsg["multiline"] = mConfig.multiline;
+		jsonMsg["contentTypeOverride"] = this.mConfig.iosContentTypeOverride.ToString();
```

Comment:
I would call this message "iosCOntentTypeOverride", because we want to make the usage clear. If we need an override for something that's Android-specific, it would be completely different. Maybe also add a comment that this string is duplicated in EditBox_iOS.m.

Reply:
Fixed.

Reply:
Oh right not the string duplication warning, but i think that's pretty much implied by it being a json?